.nvmrc

```diff
@@ -0,0 +1 @@
+14.17.6
```

Comment:
I noticed this is added in few of your prs. When .nvmrc file is specified, nvm will switch to the version automatically?